### PR TITLE
Toast component

### PR DESCRIPTION
**Pull request by prath-dev**
Added Toast Component .
Live Preview : http://127.0.0.1:5501/index.html#toast

### CODE REVIEW

File: components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.success-Toast {
+    background-color: green;
+    margin-left: 2rem;
```
Comment (Asuraking1n) [inline components/Toast/toast.css:3]: Here you used margin right and bottom separately you can use all at once

**Review by Asuraking1n — COMMENTED** [review]
I guess toast should be functional

File: components/Toast/toast.css (hunk 1 above)
Comment (prath-dev) [reply components/Toast/toast.css:3]: okkk thansk for your suggestion

File: components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.warning-Toast {
+    text-align: center;
+    background-color: #ffb303;
+    margin-bottom: 1rem;
+    height: 3.5rem;
+    margin-left: 2rem;
+    width: 29rem;
+    border: 2px solid #ffb303;
```
Comment (harshitpaliwal95) [inline components/Toast/toast.css:31]: u should use color variables

**Review by harshitpaliwal95 — COMMENTED** [review]
overall looks good

File: components/Toast/toast.css (hunk 2 above)
Comment (prath-dev) [reply components/Toast/toast.css:31]: okkkk thansk for suggestion

**Review by Vinayak-YB — APPROVED** [review]
The code looks good. The component is also fine

**Comment by prath-dev** [comment]
> I guess toast should be functional

okk

**Comment by prath-dev** [comment]
@harshitpaliwal95 : thanks  for your suggestion i will try to implement in second version of component library

**Pull request merged**